ingest/indexer/service/indexer_streaming_service.go

```diff
@@ -75,7 +75,7 @@ func (s *indexerStreamingService) ListenEndBlock(ctx context.Context, req types.
 		var err error
 
 		// Ingest the initial data
-		s.keepers.BankKeeper.IterateTotalSupplyWithOffsets(sdkCtx, func(coin sdk.Coin) bool {
+		s.keepers.BankKeeper.IterateTotalSupply(sdkCtx, func(coin sdk.Coin) bool {
```

Comment:
Note: to reviewer: the idea is that we always publish raw supplies (no offset) and then also publish non-zero supply offsets via a separate topic.

We would have to then do transformations downstream